packages/x-tree-view/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-tree-view",
-  "version": "7.2.0",
+  "version": "7.3.0",
```

Comment:
Don't forget to update the `@mui/x-tree-view` version and reference. 🙈 
It's currently a private package and thus is not handled by lerna...